.github/workflows/alpha-npm.yml

```diff
@@ -21,15 +21,16 @@ jobs:
 
     env:
       # Packages not listed here will be excluded from publishing
-      PACKAGES: "agent api common credentials crypto crypto-aws-kms dids identity-agent proxy-agent user-agent"
+      # These are currently in a specific order due to dependency requirements
+      PACKAGES: "crypto crypto-aws-kms common dids credentials agent identity-agent proxy-agent user-agent api"
```

Comment:
At some point this workflow stopped being able to build. After the step `Generate alpha prerelease and bump package.json @web5/* versions` modifies the package versions, pnpm is running the builds in alphabetical order which is causing dependency issues (looking for a version that doesn't exist).

A temporary remedy is to build the packages sequentially given the order in which they depend on each other.